nixos/modules/hardware/video/nvidia.nix

```diff
@@ -285,7 +285,7 @@ in
             KERNEL=="nvidia_uvm", RUN+="${pkgs.runtimeShell} -c 'mknod -m 666 /dev/nvidia-uvm-tools c $$(grep nvidia-uvm /proc/devices | cut -d \  -f 1) 1'"
           '';
           hardware.opengl = {
-            extraPackages = [ nvidia_x11.out ] ++ (lib.optional (builtins.hasAttr "libXNVCtrl" nvidia_x11.settings) nvidia_x11.settings.libXNVCtrl);
```

Comment:
This seems redundant since it was added in https://github.com/NixOS/nixpkgs/pull/316516.

Reply:
It was actually added in #312811 and then fixed in #316516, and then I realised this is not necessary, which is the reason for this PR.

Reply:
Oh wow sorry my bad!! This looks good to me!